front/assets/style/variables.scss

```diff
@@ -0,0 +1,5 @@
+$extra-small-screen: 'only screen and (max-width : 600px)';
+$small-screen: 'only screen and (max-width : 960px)';
+$medium-screen: 'only screen and (max-width : 1264px)';
+$large-screen: 'only screen and (max-width : 1904)';
+$extra-large-screen: 'only screen and (min-width : 1904)';
```

Comment:
there is already classes for this in vuetify. Can u check if u can reause them instead of reintroduce new?
https://vuetifyjs.com/en/framework/display

Reply:
@aldarund I tried to use them first, we are still using vuetify's display settings. But in scss we can not use these properties, they are accessible inside markups only as far as I've understood.